### PR TITLE
Kick user if it has no username

### LINKED ISSUE #28: Identificazione e kick degli utenti scammer di criptovalute
Sarebbe utile un meccanismo che identifichi ed espella dal gruppo utenti legati alla compravendita (spesso truffaldina) di criptovalute. Si potrebbe implementare un meccanismo di ricerca di determinate parole all'interno della Bio e dello username dell'utente in modo da avere un'identificazione di base (e.g. ricercando parole come "crypto", "investor", "bitcoin", ecc...).

### PR DESCRIPTION
**Pull request by iflare3g**

### CODE REVIEW

File: tg_bot/modules/welcome.py [hunk 1]
```diff
@@ -129,11 +129,15 @@ def cas_banned(userid):
 @run_async
 def new_member(bot: Bot, update: Update):
     chat = update.effective_chat  # type: Optional[Chat]
+    new_members = update.effective_message.new_chat_members
+    for new_mem in new_members:
+        if not new_mem.username:
+            chat.kick_member(new_mem.id)
```
Comment (MattiaFailla) [inline tg_bot/modules/welcome.py]: Are we sure that `chat` is an object on which we can go to apply the kick? Doesn't it have to be something done by the bot itself (like  `bot.restrict_chat_member()`)?

Comment (iflare3g) [reply tg_bot/modules/welcome.py:137]: I can check again the documentation of python-telegram-bot but it should work with chat object

Comment (iflare3g) [reply tg_bot/modules/welcome.py]: > Are we sure that `chat` is an object on which we can go to apply the kick? Doesn't it have to be something done by the bot itself (like  `bot.restrict_chat_member()`)?

https://python-telegram-bot.readthedocs.io/en/stable/telegram.chat.html?highlight=kick_member#telegram.Chat.kick_member
It's a shortcut of bot.kick_member
What we should check is the Note section.

**Review by MattiaFailla — APPROVED** [review]

**Comment by MattiaFailla** [comment]
@Kavuti Could you please left a review so we can merge this PR?

File: tg_bot/modules/welcome.py [hunk 2]
```diff
@@ -129,118 +130,127 @@ def cas_banned(userid):
...
-                            "ERROR in CAS banning user %s in chat %s (%s) due to %s",
-                            user_id,
-                            chat.title,
-                            chat.id,
-                            excp.message,
-                        )
-                        message.reply_text(
-                            "Diamine, non riesco a bannare questo utente."
-                        )
+                chat.kick_member(user_id, until_date=1602201900)
```
Comment (Kavuti) [inline tg_bot/modules/welcome.py]: `until_date` dovrebbe essere dinamico

**Review by Kavuti — CHANGES_REQUESTED** [review]
Bisogna modificare la `until_date` del kick che attualmente è hard-coded ad una data precisa

File: tg_bot/modules/welcome.py (hunk 2 above)
Comment (MattiaFailla) [reply tg_bot/modules/welcome.py]: `until_date` prende in input tempo unix, dovrebbe essere calcolato nel seguente modo se vogliamo sbloccare dopo cinque minuti:

`until_date=time.time() + 300`

Comment (iflare3g) [reply tg_bot/modules/welcome.py]: forse non ho pushato, devo ricontrollare

**Comment by iflare3g** [comment]
Adesso dovrebbe essere corretto @Kavuti , @MattiaFailla

File: tg_bot/modules/welcome.py (hunk 2 above)
Comment (iflare3g) [reply tg_bot/modules/welcome.py]: Aggiornato!

**Review by Kavuti — APPROVED** [review]
Ottimo lavoro

**Pull request merged**